src/algokit/cli/goal.py

```diff
@@ -47,7 +47,7 @@ def goal_command(*, console: bool, goal_args: list[str]) -> None:
             "See https://docs.docker.com/get-docker/ for more information."
         ) from ex
 
-    sandbox = ComposeSandbox()
+    sandbox = ComposeSandbox(dir_name="sandbox")
```

Comment:
I think we may need to think through this a bit more, as it's possible the running localnet is not the default one.